project-metrics/metrics_service/cron.yaml

```diff
@@ -2,3 +2,15 @@ cron:
 - description: "Update dummy metric."
   url: /_cron/recompute/IdentityMetric
   schedule: every 6 hours
+
+- description: "Update presubmit-ignored metric."
```

Comment:
should we be consistent with naming here? you changed `Presubmit-Ignored` to `"Presubmit Ignored` below

Reply:
In the cron descriptions, I'm using the identifiers from the design doc; in the module comments, I'm using the "prose" metric names. Seemed reasonable, but if it bothers you I can patch it.